cmd/starsd/cmd/root.go

```diff
@@ -71,6 +71,12 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	authclient.Codec = encodingConfig.Marshaler
 
+	// cfg := sdk.GetConfig()
+	// cfg.Seal()
```

Comment:
It's annoying that we have comment / uncomment this.

Reply:
Thankfully is a one-time thing!